domain/src/main/kotlin/no/nav/su/se/bakover/domain/vedtak/Vedtak.kt

```diff
@@ -124,13 +126,21 @@ sealed class Vedtak : VedtakFelles, Visitable<VedtakVisitor> {
     ) : Vedtak() {
 
         override fun journalfør(journalfør: () -> Either<KunneIkkeJournalføreOgDistribuereBrev.KunneIkkeJournalføre.FeilVedJournalføring, JournalpostId>): Either<KunneIkkeJournalføreOgDistribuereBrev.KunneIkkeJournalføre, EndringIYtelse> {
-            return journalføringOgBrevdistribusjon.journalfør(journalfør)
-                .map { copy(journalføringOgBrevdistribusjon = it) }
+            return if (vedtakType == VedtakType.SØKNAD || vedtakType == VedtakType.ENDRING || vedtakType == VedtakType.OPPHØR) {
```

Comment:
Vurdert å bruke `sendBrev` til denne sjekken?

Reply:
Feltet `skalFøreTilBrevUtsending` gjelder bare for IngenEndring.